src/main/resources/lang/english.lang

```diff
@@ -43,7 +43,6 @@ skript command:
 			description: Check for updates, read the changelog, or download the latest version of Skript
```

Comment:
You missed the command description

Reply:
😭

Reply:
i can't really do it for other languages, so i guess it'll just have to be english